db/migrate/20220223095704_remove_col3_from_miq_widget_set.rb

```diff
@@ -0,0 +1,26 @@
+class RemoveCol3FromMiqWidgetSet < ActiveRecord::Migration[6.0]
+  class MiqWidgetSet < ActiveRecord::Base
+    include ActiveRecord::IdRegions
+    self.table_name = "miq_sets"
+  end
+
+  def up
+    say_with_time("Moving col3 widgets to col2 and removing col3") do
+      MiqWidgetSet.all.each do |item|
+        data = eval(item.set_data)
```

Comment:
Instead, I think we can add `serialize :set_data` to the MiqWidgetSet model defined above.  Then this will just be 
```suggestion
        data = item.set_data
```